docs/cudf/source/api_docs/general_functions.rst

```diff
@@ -26,13 +26,23 @@ Top-level conversions
    :toctree: api/
 
     cudf.to_numeric
+    cudf.from_dataframe
     cudf.from_dlpack
     cudf.from_pandas
 
-Top-level dealing with datetimelike
------------------------------------
+Top-level dealing with datetimelike data
```

Comment:
Updated to match pandas headings: https://pandas.pydata.org/docs/reference/general_functions.html